sbank/waveforms.py

```diff
@@ -824,28 +821,16 @@ def from_dict(cls, params, idx, bank):
 
     def to_sngl(self):
         # All numerical values are initiated as 0 and all strings as ''
-        row = SnglInspiralTable()
-        row.mass1 = self.m1
-        row.mass2 = self.m2
-        row.mtotal = self.m1 + self.m2
-        row.mchirp = self._mchirp
-        row.eta = row.mass1 * row.mass2 / (row.mtotal * row.mtotal)
-        row.tau0, row.tau3 = m1m2_to_tau0tau3(self.m1, self.m2, self.flow)
-        row.template_duration = self.dur
+        row = super(PrecessingSpinTemplate, self).to_sngl()
```

Comment:
This can be `row = super().to_sngl()` we don't need to support python2 any more, so don't need the rest.